FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
- - opening_brace
+ - todo
+
+opt_in_rules:
+ - anyobject_protocol
+ - array_init
+ - attributes
+ - closure_body_length
+ - closure_end_indentation
+ - closure_spacing
+ - collection_alignment
+ - conditional_returns_on_newline
+ - contains_over_filter_count
+ - contains_over_filter_is_empty
+ - contains_over_first_not_nil
+ - contains_over_range_nil_comparison
+ - convenience_type
+ - discouraged_optional_boolean
+ - discouraged_optional_collection
+ - empty_collection_literal
+ - empty_count
+ - empty_string
+ - empty_xctest_method
+ - enum_case_associated_values_count
+ - expiring_todo
+ - explicit_init
+ - explicit_self
+ - extension_access_modifier
+ - fatal_error_message
+ - file_header
+ - file_name_no_space
+ - first_where
+ - flatmap_over_map_reduce
+ - force_unwrapping
+ - function_default_parameter_at_end
+ - identical_operands
+ - implicit_return
+ - implicitly_unwrapped_optional
+ - indentation_width
```

Comment:
`indentation_width` not always show correct warning (e.g. https://github.com/realm/SwiftLint/issues/3046).
Maybe it's better to be off, anyway Xcode helps to set correct indentation with `⌃i`.

Reply:
The example, there seems to be incorrect, so the warning looks legitimate